library/src/main/java/com/novoda/downloadmanager/RoomBatch.java

```diff
@@ -6,7 +6,8 @@
 import android.arch.persistence.room.PrimaryKey;
 import android.support.annotation.NonNull;
 
-@Entity(indices={@Index("batch_id")})
+@SuppressWarnings("checkstyle:visibilitymodifier") // Accessors will add a lot of boilerplate code.
```

Comment:
Can add the accessors but it seems unnecessary to me given that this is only used internally and it is super easy to read atm.